dev/planning-example.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# ============================
+# Observation Planning Toolbox
+# ============================
+"""
+The observation planning toolbox consists of a number of entities that
+simplify the planning and scheduling of observations by abstracting much
+of the boilerplate code necessary to accomplish the same thing using
+core astropy or ephemerides objects.  These can be thought of as common
+idioms or convenience classes and functions that would be recreated over
+and over by astronomers and or software engineers building telescope
+scheduling software.
+
+Important assumptions:
+
+- It should be convenient to specify dates and times in the local timezone
+  of the observer.
+
+- It should be convenient to do timezone conversions easily.
+
+  This means methods that can take dates can either take astropy
+  dates (as used internally) or python datetime objects (not naive, but
+  tagged with timezone).  So dates can be passed in as timezone-aware
+  datetime objects and results can be easily converted between such.
+
+"""
+
+# ================
+# Observer objects
+# ================
+"""
+An observer defines a single terrestrial location and applicable
+environmental attributes thereof that affect observation calculations
+at that location.
+"""
+# Define an observer by full specification
+#
+# Because there may be many attributes for an observer, I suggest using
+# keyword parameters for most of them.
+from astroplan.entity import Observer
```

Comment:
I agree with @cdeil here: i.e. ``from astroplan import Observer``

Another example of how this works can be found in ``astropy.coordinates``: https://github.com/astropy/astropy/blob/master/astropy/coordinates/__init__.py

The init just does ``from <subpkg> import *`` and that brings in everything in the ``__all__`` attribute of the subpackages.